planemo/galaxy_config.py

```diff
@@ -354,7 +355,8 @@ def _ensure_galaxy_repository_available(ctx):
         shell("git --git-dir %s fetch >/dev/null 2>&1" % gx_repo)
     else:
         remote_repo = "https://github.com/galaxyproject/galaxy"
-        shell("git clone --bare %s %s" % (remote_repo, gx_repo))
+        command = git.command_clone(ctx, remote_repo, gx_repo, bare=True)
+        shell(command)
```

Comment:
Assuming we are only targeting github/bitbucket it would be faster to pull the latest tarball. With all the test data over time a git pull can become a bottle-neck.